DQM/src/DQM/SampleValidation.cxx

```diff
@@ -0,0 +1,130 @@
+#include "DQM/SampleValidation.h"
+#include "SimCore/Event/SimParticle.h"
+#include "SimCore/Event/SimTrackerHit.h"
+#include "Framework/NtupleManager.h"
+#include <iostream>
+#include <fstream>
+#include <algorithm>
+
+namespace dqm {
+
+  void SampleValidation::configure(framework::config::Parameters& ps) {
+
+    return;
+  }
+
+  void SampleValidation::analyze(const framework::Event& event) {
+
+    //Grab the SimParticle Map and Target Scoring Plane Hits
+    auto targetSPHits(event.getCollection<ldmx::SimTrackerHit>("TargetScoringPlaneHits"));
+    auto particle_map{event.getMap<int, ldmx::SimParticle>("SimParticles")};
+
+    std::vector<int> primary_daughters;
+
+    double hard_thresh = 2500;
```

Comment:
Why are we defining hard_thresh down here and redefining it later?

Reply:
I needed to declare hard_thresh outside the first for loop since it is used in later loops. But in principle I should only declare it here without assigning it a value yet. This was likely a remnant from when hard_thresh was a hard-coded value. I can remove the value definition but will still keep the variable declared here.